resources/dynamodb/dynamodb-backup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: dynamodb.services.k8s.aws/v1alpha1
+kind: Backup
+metadata:
+  name: $BACKUP_NAME #Change the Backup Name
```

Comment:
Do we really need all the comments saying to `Change the XXX`? I would think the wording above to modify any property identified by a dollar sign and all caps is good enough.

Reply:
Thanks for the comment @jaypipes ! I've removed the comments from all of the YAML files in the original repository @RedbackThomson you can check it out.